datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/VersionController.java

```diff
@@ -93,9 +121,14 @@ public void writeVersionedDocument(@RequestHeader String user,
      * deletes latest version of file from user's private space.
      */
     @DeleteMapping("/versioned/{path:.*}")
+    @ApiOperation("Delete latest document from user's private space")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Document successfully deleted"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed